src/views/Instance/data.js

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import axios from 'axios';
+import { Auth } from '@cosmotech/core';
+import {
+  DEFAULT_EDGE_STYLE,
+  DEFAULT_NODE_STYLE,
+  DEFAULT_SELECTED_EDGE_STYLE,
+  DEFAULT_SELECTED_NODE_STYLE,
+} from './styleCytoViz';
+import { ORGANIZATION_ID, WORKSPACE_ID } from '../../config/AppInstance';
+import instanceViewData from '../../config/InstanceVisualization.js';
+
+const IS_INSTANCE_VIEW_FUNCTION_CONFIG_VALID = !(instanceViewData.dataSource == null);
+export { IS_INSTANCE_VIEW_FUNCTION_CONFIG_VALID };
+
+const _formatLabelWithNewlines = (label) => label?.replace(/[_|\s]/g, '\n') || '';
+
+const _forgeCytoscapeNodeData = (node, classes, nodesGroupMetadata = {}, nodesParentsDict = {}) => {
+  const parent = nodesParentsDict[node.id];
+  const { style, ...otherMetadata } = nodesGroupMetadata;
+  return {
+    group: 'nodes',
+    data: { ...node, label: _formatLabelWithNewlines(node.id), parent: parent },
+    classes: classes,
+    ...otherMetadata,
+  };
+};
+
+const _forgeCytoscapeEdgeData = (edge, classes, edgesGroupMetadata = {}) => {
+  const { style, ...otherMetadata } = edgesGroupMetadata;
+  return {
+    group: 'edges',
+    data: edge,
+    classes: classes,
+    ...otherMetadata,
+  };
+};
+
+export const _processGraphCompounds = (dataset, compoundsGroups) => {
```

Comment:
is export necessary here if this function isn't used elsewhere ?

Reply:
Nice catch, classic copy/paste mistake :smile: